apacheconfig/wloader.py

```diff
@@ -8,10 +8,68 @@
 
 import abc
 import six
+import contextlib
+
+from apacheconfig.lexer import make_lexer
+from apacheconfig.parser import make_parser
+from apacheconfig.reader import LocalHostReader
 
 from apacheconfig import error
 
 
+@contextlib.contextmanager
+def make_writable_loader(**options):
```

Comment:
Should we move `make_writable_loader` to `apacheconfig/__init__`? Could we reuse `make_loader` that already exists there?

Reply:
Good idea! Just re-used existing `make_loader`.